amgut/db/patches/0018.sql

```diff
@@ -0,0 +1,263 @@
+INSERT INTO ag.survey_question_response (survey_question_id, response, display_index) VALUES (146, 'Trinidad and Tobago', 221);
+INSERT INTO ag.survey_question_response (survey_question_id, response, display_index) VALUES (146, 'Tunisia', 222);
+INSERT INTO ag.survey_question_response (survey_question_id, response, display_index) VALUES (146, 'Turkey', 223);
+INSERT INTO ag.survey_question_response (survey_question_id, response, display_index) VALUES (146, 'Turkmenistan', 224);
+INSERT INTO ag.survey_question_response (survey_question_id, response, display_index) VALUES (146, 'Turks and Caicos Islands', 225);
+INSERT INTO ag.survey_question_response (survey_question_id, response, display_index) VALUES (146, 'Tuvalu', 226);
+INSERT INTO ag.survey_question_response (survey_question_id, response, display_index) VALUES (146, 'Uganda', 227);
+INSERT INTO ag.survey_question_response (survey_question_id, response, display_index) VALUES (146, 'Ukraine', 228);
+INSERT INTO ag.survey_question_response (survey_question_id, response, display_index) VALUES (146, 'United Arab Emirates', 229);
+INSERT INTO ag.survey_question_response (survey_question_id, response, display_index) VALUES (146, 'United Kingdom', 230);
+INSERT INTO ag.survey_question_response (survey_question_id, response, display_index) VALUES (146, 'United States', 231);
+INSERT INTO ag.survey_question_response (survey_question_id, response, display_index) VALUES (146, 'United States Minor Outlying Islands', 232);
+INSERT INTO ag.survey_question_response (survey_question_id, response, display_index) VALUES (146, 'Uruguay', 233);
+INSERT INTO ag.survey_question_response (survey_question_id, response, display_index) VALUES (146, 'Uzbekistan', 234);
+INSERT INTO ag.survey_question_response (survey_question_id, response, display_index) VALUES (146, 'Vanuatu', 235);
+INSERT INTO ag.survey_question_response (survey_question_id, response, display_index) VALUES (146, 'Venezuela', 236);
+INSERT INTO ag.survey_question_response (survey_question_id, response, display_index) VALUES (146, 'Viet Nam', 237);
+INSERT INTO ag.survey_question_response (survey_question_id, response, display_index) VALUES (146, 'Virgin Islands, British', 238);
+INSERT INTO ag.survey_question_response (survey_question_id, response, display_index) VALUES (146, 'Virgin Islands, U.S.', 239);
+INSERT INTO ag.survey_question_response (survey_question_id, response, display_index) VALUES (146, 'Wallis and Futuna', 240);
+INSERT INTO ag.survey_question_response (survey_question_id, response, display_index) VALUES (146, 'Western Sahara', 241);
+INSERT INTO ag.survey_question_response (survey_question_id, response, display_index) VALUES (146, 'Yemen', 242);
+INSERT INTO ag.survey_question_response (survey_question_id, response, display_index) VALUES (146, 'Zambia', 243);
+INSERT INTO ag.survey_question_response (survey_question_id, response, display_index) VALUES (146, 'Zimbabwe', 244);
+
+-- add country column to zipcode table
```

Comment:
yes, I pulled that out as a separate part because it is needed regardless of if this goes in or not. I'll update this pull request again once we have IRB approval.